ecommers/lib/ui/pages/more_page.dart

```diff
@@ -51,53 +53,54 @@ class MorePage extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    return SingleChildScrollView(
-      scrollDirection: Axis.vertical,
-      child: Column(
-        mainAxisAlignment: MainAxisAlignment.start,
-        crossAxisAlignment: CrossAxisAlignment.start,
-        children: <Widget>[
-          Padding(
-            padding: const EdgeInsets.symmetric(horizontal: Insets.x6),
-            child: Text(
-              localization.morePage,
-              style: textTheme.headline6,
+    return BasePage(
+      createProvider: () => MoreProviderModel(),
+      child: SingleChildScrollView(
+        scrollDirection: Axis.vertical,
+        child: Column(
+          mainAxisAlignment: MainAxisAlignment.start,
+          crossAxisAlignment: CrossAxisAlignment.start,
+          children: <Widget>[
+            Padding(
+              padding: const EdgeInsets.symmetric(horizontal: Insets.x6),
+              child: Text(
+                localization.morePage,
+                style: textTheme.headline6,
+              ),
             ),
-          ),
-          const SizedBox(height: 35.0),
-          const MenuList(
-            margin: menuListMargin,
-            itemList: topMenuList,
-          ),
-          const SizedBox(height: 15.0),
-          const MenuList(
-            margin: menuListMargin,
-            itemList: bottomMenuList,
-          ),
-          const SizedBox(height: 40.0),
-          _buildLogOutButton(),
-          const SizedBox(height: 30.0),
-        ],
+            const SizedBox(height: 35.0),
+            const MenuList(
+              margin: menuListMargin,
+              itemList: topMenuList,
+            ),
+            const SizedBox(height: 15.0),
+            const MenuList(
+              margin: menuListMargin,
+              itemList: bottomMenuList,
+            ),
+            const SizedBox(height: 40.0),
+            _buildLogOutButton(),
+            const SizedBox(height: 30.0),
+          ],
+        ),
       ),
     );
   }
 
   Widget _buildLogOutButton() {
-    return Center(
-      child: CupertinoButton(
-        onPressed: logOutPressHandler,
-        child: Text(
-          localization.logOut,
-          style: textTheme
-              .headline5
-              .apply(color: BrandingColors.primary),
-        ), //TODO use proovider
-      ),
+    return Consumer<MoreProviderModel>(
```

Comment:
do u really need consumer here? Because u use it just for getting provider object.

Reply:
remove consumer